### DISCUSSION TITLE
Best way to disable rules by severity?

### DISCUSSION THREAD

**Discussion by xxvs** [Security-Onion-Solutions/securityonion#7388] [Unsupported Versions] [answered]
I've read the documentation on managing alerts and rules. But I'm curious as to what would be the most efficient and least resource intensive way to completely ignore low severity alerts.

I'd imagine that it would be better to disable them rather than suppress, but what would be the best way of doing that?

**Comment by dougburks** [comment, marked as answer]
Many organizations disable entire categories of rules:
https://docs.securityonion.net/en/2.3/managing-alerts.html#identifying-rule-categories

**Reply by xxvs** [reply]
I'm aware of this. I was just wondering if it's possible to disable all rules that show up as _Low_ in SO, or if I have to do that manually?

**Reply by dougburks** [reply]
Keep in mind that just because a rule writer classifies the rule as low severity doesn't necessarily mean that you or your organization will agree with that classification.

If you really want to disable by severity, you might be able to determine what classifications map to low severity and then add a regular expression to the disabled list for each of those. Regular expressions are mentioned on the page I linked above.

**Reply by xxvs** [reply]
Got it, thank you.
